package.json

```
@@ -81,13 +88,25 @@
     "parser": "@typescript-eslint/parser",
     "plugins": [
       "@typescript-eslint",
-      "prettier"
+      "prettier",
+      "testing-library",
+      "jest-dom"
     ],
     "extends": [
       "eslint:recommended",
       "plugin:@typescript-eslint/recommended",
-      "plugin:prettier/recommended"
-    ]
+      "plugin:prettier/recommended",
+      "plugin:testing-library/recommended",
+      "plugin:jest-dom/recommended",
+      "plugin:react/recommended"
+    ],
+    "rules": {
+      "jest-dom/prefer-to-have-value": "off",
+      "react/jsx-curly-brace-presence": [
+        2,
```

Nit - I find the strings easier to read than the numbers :)
```suggestion
        "error",
```